Rename TxName to fateOp for clarity (#5230)

Suvrat1629 opened the pull request:
Replaced occurrences of TxName, txName, and similar references with fateOp.

Changes
    Renamed `txName` → `fateOp` where applicable
    Updated related method names

Fixes #5230

Suvrat1629 commented:
@kevinrr888 
I had renamed all the references that I could find, please take a look and tell me if have missed anything.
Thank you!

kevinrr888 commented on core/src/main/java/org/apache/accumulo/core/fate/user/FateMutatorImpl.java:
```diff
@@ -130,7 +130,7 @@ public FateMutator<T> putUnreserveTx(FateStore.FateReservation reservation) {
 
   @Override
   public FateMutator<T> putName(byte[] data) {
```
```suggestion
  public FateMutator<T> putFateOp(byte[] data) {
```

Suvrat1629 replied:
Are there such similar method names that need to be changed?

kevinrr888 commented on core/src/main/java/org/apache/accumulo/core/fate/user/schema/FateSchema.java:
```diff
@@ -44,8 +44,8 @@ public static class TxInfoColumnFamily {
 
-    public static final String TX_NAME = "txname";
-    public static final ColumnFQ TX_NAME_COLUMN = new ColumnFQ(NAME, new Text(TX_NAME));
+    public static final String FATE_OP = "fateOp";
```
```suggestion
    public static final String FATE_OP = "fateop";
```

kevinrr888 reviewed (commented):
Changes so far look good. There may still be some references not addressed. I recommend checking `grep -riE --include *.java 'tx.?name'`. This will find any reference to `tx<0 or 1 char>name` ignoring case.

kevinrr888 commented:
Yes, comments should be updated

Suvrat1629 commented:
@kevinrr888 
I updated the comments and also since in one of the prior comments I had changed `putName` to `putFateOp` I changed all the references of `putName` to `putFateOp`. Please take a look.
